suites/pacific/rgw/tier-2_rgw_test-multisite-metadata-sync.yaml

```diff
@@ -215,6 +215,7 @@ tests:
           config:
             config-file-name: test_bucket_create_del.yaml
             script-name: test_Mbuckets_with_Nobjects.py
+            verify-io-on-site: ["ceph-sec"]
```

Comment:
We dont need to run `verify-io-on-site` parameter if we are doing any delete operation.
 `verify-io-on-site` will fail TC since the data is already deleted from one site